cmd/rookflex/cmd/root.go

```diff
@@ -99,10 +101,13 @@ func redirectStdout(client *rpc.Client, fn func() error) error {
 	}()
 
 	err := fn()
-	w.Close()
+	// #nosec G307
+	defer w.Close()
```

Comment:
We don't want to defer this `w.Close()`, just check for its error

Reply:
Done.

Reply:
still need to revert this